data/json/scenarios/scenarios.json

```diff
@@ -101,6 +101,16 @@
     "professions" : ["unemployed","bionic_patient","patient","broken_cyborg"],
     "flags" : ["CHALLENGE"]
   },{
+    "type" : "scenario",
+    "ident" : "prisonbreak",
+    "name" : "Prison Break",
+    "points" : -4,
+    "description" : "You were a convicted felon planning your escape before the event.  Now the prisoners have turned into mindless horrors and the security bots are shooting on sight... you might need to expedite your escape plans.",
```

Comment:
It might be good to replace set of three points ("...") by a proper ellipsis symbol ("…").